### ISSUE TITLE
Crafttweaker Support

### ISSUE THREAD

**Issue by Omegapsychosis** [kitsushadow/forgecraft-issues#7]
This mod is awesome, but I was wondering if it would be possible to get crafttweaker support for things like the ToolBench or the Anvil.

I am working on a custom pack that utilizes a lot of tweaked recipes and a pseudo-class system,  and I would love the ability to use your mechanics to enhance my blacksmith perks.

**Comment by Rsslone** [comment]
+1

This could be a very powerful feature.

**Comment by gatoborrachon** [comment]
yeah it would be amazing, there are a lot of posibilities with the 4 machines, Forging, Anvil crafting, Casting and Tool Bench

**Comment by winterSteve25** [comment]
Yes please this would be so useful
